src/scale-manager/ScaleManager.js

```diff
@@ -29,11 +29,13 @@ export class ScaleManager {
     const width = event.target.innerWidth;
     const height = event.target.innerHeight;
 
-    this.callback({
-      width,
-      height,
-      ratio: width / height
-    });
+    setTimeout(() => {
+      this.callback({
+        width,
+        height,
+        ratio: width / height
```

Comment:
This isn't going to fix the issue by the way. You're going to have to recalculate the ratio:

```
setTimeout(() => {
  this.callback({
    width: window.innerWidth,
    height: window.innerHeight,
    ratio: window.innerWidth / window.innerHeight
  })
}, 500)
```

since the `window.innerWidth` and `window.innerHeight` values change on the fly in iOS

Reply:
oh, oops, that whole block should be in the time out. good catch.